<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab_01</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>351</width>
            <height>491</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
           <property name="spacing">
            <number>-1</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QTableWidget" name="table_points">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="dragDropOverwriteMode">
              <bool>false</bool>
             </property>
             <property name="rowCount">
              <number>0</number>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>165</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <column>
              <property name="text">
               <string>X</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Y</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>80</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>339</width>
                <height>80</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="font">
                      <font>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>X:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="input_X">
                     <property name="font">
                      <font>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="maximum">
                      <double>9999999999.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="font">
                      <font>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Y:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="input_Y">
                     <property name="font">
                      <font>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="maximum">
                      <double>9999999999.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="btn_add_point">
                 <property name="text">
                  <string>Добавить точку</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>80</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>331</width>
                <height>80</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Номер:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="input_index">
                     <property name="font">
                      <font>
                       <pointsize>16</pointsize>
                      </font>
                     </property>
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>999999999</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="btn_del_point">
                 <property name="text">
                  <string>Удалить точку</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_del_points">
             <property name="text">
              <string>Удалить все точки</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>9</y>
            <width>351</width>
            <height>71</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="btn_task">
             <property name="text">
              <string>Условие задачи</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_get_result">
             <property name="text">
              <string>Найти ответ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="title">
          <string>Ответ</string>
         </property>
         <widget class="QLabel" name="lable_answer">
          <property name="geometry">
           <rect>
            <x>7</x>
            <y>25</y>
            <width>351</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="graphic_lable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>690</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
